metadata/trackers.json

```diff
@@ -26,7 +26,8 @@
         },
         "redact_exemption" : {
             "keywords:.*WebKit Security Does Not Apply to Shipping" : true,
-            "keywords:.*WebKit Security Merge-Back Approved" : true
+            "keywords:.*WebKit Security Merge-Back Approved" : true,
+            "keywords:.*WebKit Security Approved To Land in Open Source" : true
```

Comment:
Is this case-sensitive? I think it's right to have lowercase "in" but I want to make sure it won't trip us up.

Reply:
It is case sensitive, yes.